packages/flutter/lib/src/cupertino/picker.dart

```diff
@@ -344,16 +342,16 @@ class _RenderCupertinoPickerSemantics extends RenderProxyBox {
     this.controller = controller;
   }
 
-  FixedExtentScrollController get controller => _controller;
-  FixedExtentScrollController _controller;
-  set controller(FixedExtentScrollController value) {
+  FixedExtentScrollController? get controller => _controller;
+  FixedExtentScrollController? _controller;
+  set controller(FixedExtentScrollController? value) {
     if (value == _controller)
       return;
     if (_controller != null)
-      _controller.removeListener(_handleScrollUpdate);
+      _controller!.removeListener(_handleScrollUpdate);
     else
-      _currentIndex = value.initialItem ?? 0;
-    value.addListener(_handleScrollUpdate);
+      _currentIndex = value!.initialItem;
+    value!.addListener(_handleScrollUpdate);
```

Comment:
I don't understand how value is guaranteed to be non-null here.

Reply:
I change the code to make controller non-nullable.